sdk/typescript/README.md

```diff
@@ -40,6 +40,74 @@ const gatewayApi = GatewayApiClient.initialize({
 const { status, transaction, stream, state } = gatewayApi
 ```
 
+## Connecting to a secured gateway using authorization header
+
+This example shows how to provide basic auth, JWT and Bearer Token can be done the same way by passing the authorization header.
+
+To work with this set-up you will need to:
+* Ensure your `basePath` is set up to connect to the correct host and port, on which Gateway API. is exposed.
+* Set the `Authorization` header on your request, configured with your basic auth credentials for the Gateway API.
+* In the `agent` / `dispatcher` configuration, include the self-signed certificate by using the `ca` parameter, or if you understand the implications and have precautions against MITM, use `rejectUnauthorized: false` with the https agent.
+
+### With `node-fetch`
+
+An example, with the `rejectUnauthorized: false` line, assuming you are EG on localhost, or a private network where MITM attacks are mitigated.
+
+```typescript
+import https from "node:https";
+import fetch from "node-fetch"
+import https from "node:https";
+import { GatewayApiClient, RadixNetwork } from '@radixdlt/babylon-gateway-api-sdk'
+
+const basicAuthUsername = "????";
+const basicAuthPassword = "????"; // From your gateway set-up - provide this securely to your application
+
+const gatewayApiClient = GatewayApiClient.initialize({
+  networkId: RadixNetwork.Mainnet,
+  applicationName: 'Your dApp Name',
+  applicationVersion: '1.0.0',
+  applicationDappDefinitionAddress: 'account_rdx12y4l35lh2543nfa9pyyzvsh64ssu0dv6fq20gg8suslwmjvkylejgj',
+  agent: new https.Agent({
+    keepAlive: true,
+    // NOTE - Only add the below line if you've taken precautions to avoid MITM attacks between you and the gateway api
+    rejectUnauthorized: false,
+  }),
+  headers: {
+      "Authorization": `Basic ${Buffer.from(`${basicAuthUsername}:${basicAuthPassword}`).toString("base64")}`
+  }
+});
+```
+
+### With native Node.JS `fetch`
+
+If wanting to customise the certificate settings on the request, you will need install `undici` (as per [this comment](https://github.com/nodejs/undici/issues/1489#issuecomment-1543856261)):
+
+The below includes the `rejectUnauthorized: false` line, assuming you are EG on localhost, or a private network where MITM attacks are mitigated.
```

Comment:
As above, let's get rid of this line too.

Reply:
fixed